lib/createProtocol.js

```diff
@@ -12,7 +12,7 @@ export default scheme => {
 
       readFile(path.join(__dirname, pathName), (error, data) => {
         if (error) {
-          console.error(`Failed to register ${scheme} protocol`, error)
+          console.error(`Failed to read ${pathName} on ${scheme} protocol`, error)
```

Comment:
IMO the error should also be re-thrown again here, and also when registering the protocol fails.

Reply:
@nklayman , what about re-raising one or both of these errors - it seems at least failure to register a protocol is a pretty serious failure.

wrt the read errors, I encountered those unexpectedly in production builds when opening the chrome JS console because it would fetch .js.map files, and some of them didnt exist in the production build.  So I have a more complex error handling block here which detects and ignores .js.map read failures in production, and then re-raises the error for all other read failures.

Reply:
I don't exactly understand what you mean by re-raising the error. Could you elaborate a little more?